server/validation/login.js

```diff
@@ -0,0 +1,30 @@
+const Validator = require('validator');
+const isEmpty = require('is-empty');
+
+module.exports = validateLoginInput = (data) => {
+  let errors = {};
+
+  data.email = !isEmpty(data.email) ? data.email : '';
+  data.password = !isEmpty(data.password) ? data.password : '';
```

Comment:
I don't understand the purpose of this.
The way I'm reading it is: Check if the value of this key is not empty, and if it's not, assign the value of the key to be the value that it already held, but if it is, assign it to be an empty string. 
If that is what you're trying to do, isn't it the same as simply not doing anything?
If, what you're trying to do is to convert a possible `undefined` value into an empty string, you can do that with a simple ternary operator:

```javascript
data.email = data.email ? data.email : '';
```